stactools_pipelines/cdk/app.py

```diff
@@ -3,24 +3,31 @@
 import aws_cdk as cdk
 import yaml
 
+from stactools_pipelines.cdk.jwt_cache_stack import JwtCacheStack
 from stactools_pipelines.cdk.lambda_stack import LambdaStack
 from stactools_pipelines.models.pipeline import Pipeline
 
 # Required environment variables
 pipeline = os.environ["PIPELINE"]
 stack_name = pipeline.replace("_", "-")
 
+app = cdk.App()
+
+jwt_cache_stack = JwtCacheStack(
+    app,
+    stack_name,
+)
```

Comment:
We're following a unique pattern where we have a stack per pipeline. I've created this separate stack so we don't create unique DynamoDB tables per pipeline, rather there should be a singular stack that is created and reused across pipeline deployments.